connectors/cache/fallback.go

```diff
@@ -393,13 +409,18 @@ func (c *Connector) cacheWrite(w func() error) error {
 	return nil
 }
 
+func (c *Connector) shouldSkipInvalidateCacheOnWrite(ei *dosa.EntityInfo) bool {
+	if c.skipWriteInvalidateEntitiesMap == nil {
```

Comment:
don't need this check, it will just return false for nil map

Reply:
It only returns false because of the struct initialization.